test/src/api/expect/testExpectTitle.js

```diff
@@ -44,8 +44,8 @@ describe('expect.title', function() {
     this.client.api.globals.waitForConditionPollInterval = 9;
     this.client.api.globals.abortOnAssertionFailure = true;
 
-    Nocks.title('hp vasq');
-    Nocks.title('hp vasq');
+    Nocks.title('hp vasq').title('hp vasq').title('hp vasq').title('hp vasq');
```

Comment:
Just wanted to understand, why this change is necessary? By change, I meant extra chaining of `title`?

Reply:
Did this in hopes of fixing a intermittent bug. But this did not fix the issue, needs deeper analysis. Basically the whenever there is  `waitForConditionTimeout` and `waitForConditionPollInterval` defined, the expected number of retries does not match the intended number. In this case the number exceeds, and causes nocks to throw error. Hence added this extra `title` as buffer.